openedx/adg/lms/applications/tests/test_models.py

```diff
@@ -117,16 +117,58 @@ def test_user_application_string_representation(user_application):
     assert expected_str == actual_str
 
 
+@pytest.fixture(name='current_time')
+def current_datetime():
+    return datetime.now()
```

Comment:
Can you please move it to some common place may be in root/conftest.py. and make sure it won't effect any other tests. may be edx already have a fixture with same name.